include/common/nas_headers.h

```diff
@@ -68,12 +72,13 @@ enum eps_nas_mesage_type {
 	DetachAccept = 0x46,
 	DetachRequest = 0x45,
 	TauAccept    = 0x49,
-    TauReject    = 0x4b,
+    	TauReject    = 0x4b,
```

Comment:
Can you add following at the start of this enum

/* Refer - 24.301 , Table - 9.8.1 */

Reply:
Done.